src/development/LocalTest/Controllers/Events/EventsController.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Threading.Tasks;
+
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Repository;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Provides operations for handling events
+    /// </summary>
+    [Route("events/api/v1/events")]
```

Comment:
changed to events/api/v1/app in the platform, should be changed here as well

Reply:
Updating.